tests/foreman/api/test_multiple_paths.py

```diff
@@ -252,7 +253,7 @@ def test_negative_post_unauthorized(self):
         )
         for entity_cls in set(valid_entities()) - set(exclude_list):
             with self.subTest(entity_cls):
-                logger.debug('test_post_unauthorized arg: %s', entity_cls)
+                self.logger.debug('test_post_unauthorized arg: %s', entity_cls)
```

Comment:
@rplevka : hey.. I see that we changed the `debug` log to `info` everywhere however not at this place ?